Data-Collection-Inventory/2021Q1_csvs/grant.csv

```diff
@@ -1,32 +1,20 @@
 grant_reference_id,grant_title,grant_identifier,funding_agency_reference_id,awardee_reference_id,report_symbol
 Allen Institute funder,Allen Institute,Allen Institute,Allen Institute for Brain Science,Allen Institute for Brain Science,
-Autism_EC grant,No title,supplement  to  #4987,Autism Speaks,University of California at San Diego,
-BrainSpan_EL_MH grant,TRANSCRIPTIONAL ATLAS OF HUMAN BRAIN DEVELOPMENT,1RC2MH089921-01,National Institute of Mental Health,Allen Institute for Brain Science,
-BrainSpan_JK grant,TRANSCRIPTIONAL ATLAS OF HUMAN BRAIN DEVELOPMENT,1RC2MH090047-01,National Institute of Mental Health,University of Southern California,
-BrainSpan_NS grant,TRANSCRIPTIONAL ATLAS OF HUMAN BRAIN DEVELOPMENT,1RC2MH089929-01,National Institute of Mental Health,Yale University,
-R24 Ghosh grant,DANDI: Distributed Archives for Neurophysiology Data Integration,1R24MH117295-01A1,National Institute of Mental Health,Massachusetts Institute of Technology,9795271
-R24 Ropelewski grant,A Confocal Fluorescence Microscopy Brain Data Archive,1R24MH114793-01,National Institute of Mental Health,Pittsburgh Supercomputing Center,9567623
-R24 White grant,A BRAIN Initiative resource: The neuroscience multi-omic data archive,1R24MH114788-01,National Institute of Mental Health,University of Maryland,9565669
-RF1 Chung grant,"Highly specific, renewable, and cost-effective antibody toolbox for 3D proteomic phenotyping of the brain",1RF1MH121270-01,National Institute of Mental Health,Massachusetts Institute of Technology,9852828
 RF1 Mueller grant,"Accessible technologies for high-throughput, whole-brain reconstructions of molecularly characterized mammalian neurons",1RF1MH121539-01,National Institute of Mental Health,Johns Hopkins University,9868000
 RF1 Nowakowski grant,Mapping Developmental Lineage Relationships in the Cerebral Cortex,1RF1MH121268-01,National Institute of Mental Health,University of California at San Francisco,9852810
 RF1 Regev grant,Scaling up spatial RNA profiling with compressed sensing,1RF1MH121289-01,National Institute of Mental Health,Broad Institute,9852697
 RF1 Tasic grant,Cell class- or type-specific viruses for brain-wide labeling and neural circuit examination,1RF1MH121274-01,National Institute of Mental Health,Allen Institute for Brain Science,9852868
 RF1 Tilgner grant,Single Cell Isoform Expression Across Mouse Brain Regions and Development,1RF1MH121267-01,National Institute of Mental Health,Weill Medical College of Cornell University,9852789
-SFARI_EC grant,No title,SFARI-07-08s,Simons Foundation,University of California at San Diego,
-U01 Chung grant,Towards Integrated 3D Reconstruction of Whole Human Brains at Subcellular Resolution,1U01MH117072-01,National Institute of Mental Health,Massachusetts Institute of Technology,9584926
```

Comment:
keep U01 Chung

Reply:
Added BICCN-related grants and subprograms that don't have associated projects yet.